ports/atmel-samd/Makefile

```diff
@@ -427,6 +427,7 @@ endif
 OBJ += $(addprefix $(BUILD)/, $(SRC_S:.s=.o))
 
 SRC_QSTR += $(SRC_C) $(SRC_SUPERVISOR) $(SRC_COMMON_HAL_EXPANDED) $(SRC_SHARED_MODULE_EXPANDED) $(STM_SRC_C)
+SRC_QSTR_PP += peripherals/samd/$(CHIP_FAMILY)/clocks.c
```

Comment:
Could you just put in a comment saying the `SRC_QSTR_PP` files are files that generate QSTRs via a preprocessor macor, or rename it to `SRC_QSTR_FROM_PREPROCESSOR` or something like that?